lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -289,12 +304,18 @@ def enable_disable_extensions(self, extensions: Dict[str, bool], database: str =
                     cursor.execute("SELECT datname FROM pg_database WHERE NOT datistemplate;")
                     databases = {database[0] for database in cursor.fetchall()}
 
+            orderedExtensions = OrderedDict()
+            for plugin in DEPENDENCY_PLUGINS:
+                orderedExtensions[plugin] = extensions[plugin]
```

Comment:
```suggestion
                orderedExtensions[plugin] = extensions.get(plugin, False)
```
Creating databases passes only enabled plugins, which causes instability in the integration tests.